### PR TITLE
update website

### PR DESCRIPTION
**Pull request by mjavorka**
- update portfolio and who we are texts
- add schema.org
- add IC

### CODE REVIEW

**Comment by mjavorka** [comment]
After release schema should be validated on https://search.google.com/structured-data/testing-tool/u/0/

File: _includes/portfolio.html [hunk 1]
```diff
@@ -6,7 +6,13 @@ <h2 class="header grey">Our Work</h2>
         <div class="row">
           <div class="col-md-12">
             <p>
-                We have started just recently, but we have two projects: the first is an app <strong>Ström</strong>, that is detecting deforestation - our main project. But during the build itself, we did find out that there are not the right tools, so we also did make <strong>Sentinel Downloader</strong> - a useful tool that helps us work more efficiently - and you can use it too!
+              We like to explore opportunities and then continue with viable projects -
+              that's the reason, why we have so many of them! Have a look <br>
```
Comment (surfer19) [inline _includes/portfolio.html]: `Have a look` - I would change it or remove it (sounds not correct for me in this context)

File: _includes/portfolio.html [hunk 2]
```diff
@@ -6,7 +6,13 @@ <h2 class="header grey">Our Work</h2>
...
+              <strong>Ström</strong> is an app detecting deforestation, with future expanse into biomass and forest health analysis.<br>
```
Comment (surfer19) [inline _includes/portfolio.html]: I suggest to put each "project" to new` <p>` "section" and put little margin between them.
Also make some space under first sentence.

File: _includes/portfolio.html (hunk 1 above)
Comment (Vasho) [inline _includes/portfolio.html]: change the line to: "that's the reason, why we have so many projects!<br>

Comment (mjavorka) [reply _includes/portfolio.html]: updated

**Pull request merged**